docs/config.yaml

```
@@ -113,17 +115,38 @@ s3:
 # OPTIONAL: container registry-related keys
 registry_repos:
   # OPTIONAL: repo to which to push oscontainer
-  oscontainer: quay.io/fedora/fedora-coreos
+  oscontainer:
+    # REQUIRED: repo to which to push oscontainer
```

Minor: and here, just

```suggestion
    # REQUIRED: repo name
```